guides/v2.1/cloud/env/variables-intro.md

```diff
@@ -65,21 +65,23 @@ return array(
    ...
 );
 ```
-
 {% include note.html type="info" content="JS bundling and JS/CSS merging do not work with SCD on demand." %}
 
 ### `SKIP_HTML_MINIFICATION`
 
--  **Default**—`false`
+-  **Default**
+   - `true` for `ece-tools` 2002.0.13 and later
+   - `false` for earlier versions of `ece-tools`
 -  **Version**—Magento 2.1.4 and later
 
-Skip copying the static view files in the `var/view_preprocessed` directory to reduce downtime when deploying to the Staging and Production environments and generates minified HTML when requested.
+Determines whether static view files in the `var/view_preprocessed` are copied at the end of the build stage and whether HTML minification of static content occurs during deployment or on request. Set this value to `true` to reduce downtime when deploying to the Staging and Production environments.
```

Comment:
Need to indicate what the `var/view_preprocessed` is...like directory.
The sentence is passive. Consider "Determines whether to copy the static view files in the `var/view_preprocessed` directory at the end of the build stage..."